### PR TITLE
EWL-1049 - Add jest emotion

### PR DESCRIPTION
**Pull request by psaint-j**
# About the type of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Test plan

- [ ] Empty data
- [ ] Invalid data
- [x] Correct data

# Checklist
### Before code review

- [x] Move Jira ticket into PR
- [ ] Check with another developers that features sounds good

### After code review

- [ ] Move Jira ticket into QA

### After QA validation
If conflicts are present in github:
  - [ ] `git pull --rebase origin master`
  - [ ] Fix conflicts
  - [ ] `git rebase --continue` repeat until there is no more conflicts
  - [ ] `git push --force-with-lease`

If there's no conflicts
- [ ] squash and merge
- [ ] delete branch

### CODE REVIEW

File: src/components/Card/ContentCard-v2.js [hunk 1]
```diff
@@ -190,11 +190,6 @@ const desktopStyles = `
     font-size: 28px;
     line-height: 36px;
     margin: 14px 40px 0 40px;
-
-    ${breakpoints.large(`
-      font-size: 28px;
-      line-height: 36px;
-    `)};
```
Comment (davidgarry) [inline src/components/Card/ContentCard-v2.js]: Why did you remove this css ?

Comment (psaint-j) [reply src/components/Card/ContentCard-v2.js]: there have code duplication on large, medium style or whatever the style still the same

Comment (psaint-j) [reply src/components/Card/ContentCard-v2.js]: And I did that because i have trouble with that line

Comment (JulienKode) [reply src/components/Card/ContentCard-v2.js]: I've try this update too and I face the same issue
